src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -313,4 +313,12 @@ Valid from:</comment>
   <data name="ErrorPackageSignatureInvalid" xml:space="preserve">
     <value>Package signature is invalid.</value>
   </data>
+  <data name="SignFailureCertificateInvalidProviderType" xml:space="preserve">
+    <value>The following certificate cannot be used for package signing as the private key provider is unsupported - {0}</value>
```

Comment:
❌ Remove all trailing `{0}`'s from localized strings where the value is a new line. The two strings are unrelated and what you are really doing here is giving the author localizing an opportunity to move `{0}` to somewhere in the middle of the string, which I would say it always going to be a mistake.

The comment here explains to the loc author what will be on the next lines, but it gives them zero control over the formatting of that content. There is no benefit that I see to this unless I'm missing something.

This needs to be fixed for ALL strings where this is going on that I see in this file, not just this one.